server/protocols/jmap-rfc-8621-integration-tests/jmap-rfc-8621-integration-tests-common/src/main/scala/org/apache/james/jmap/rfc8621/contract/MailboxSetMethodContract.scala

```diff
@@ -5517,6 +5528,7 @@ trait MailboxSetMethodContract {
   }
 
   @Test
+  @Tag(CategoryTags.BASIC_FEATURE)
```

Comment:
Remember: it is smoke tests. The ratio should be between 15% to 20% per test class.

Reply:
Yes... 11 smoke tests for a total of 111 tests (yes there is this much for Mailbox/set)... makes something around like 10% I believe? :trollface: